hypothesis_bio/hypothesis_bio.py

```diff
@@ -214,21 +215,55 @@ def cds(
 
 
 @composite
-def parsed_fasta(
-    draw, comment_source: SearchStrategy = None, sequence_source: SearchStrategy = None
-) -> Dict[str, str]:
+def fasta(
+    draw,
+    comment_source: SearchStrategy = None,
+    sequence_source: SearchStrategy = None,
+    wrap_lines=True,
+    wrap_length: Optional[int] = None,
+    allow_windows_line_endings=True,
+) -> str:
+
     if comment_source is None:
         comment_source = text(alphabet=characters(min_codepoint=32, max_codepoint=126))
     if sequence_source is None:
         sequence_source = dna()
 
     comment = draw(comment_source)
     sequence = draw(sequence_source)
-    return {
-        "fasta": ">" + comment + "\n" + sequence,
-        "comment": comment,
-        "sequence": sequence,
-    }
+
+    if not wrap_lines and wrap_length is not None:
+        raise ValueError("Can't have wrap length if not wrapping")
+
+    # the nice case where the user gave the wrap size
+    if wrap_lines and wrap_length is not None:
+        sequence = wrap(sequence, width=wrap_length)
```

Comment:
would be easier to use `textwrap.fill()` as it does all of this in one. See https://github.com/Lab41/hypothesis-bio/issues/46#issuecomment-544184451